kebechet/managers/initialization/__init__.py

```diff
@@ -0,0 +1,4 @@
+""" Initializes a YAML file for kebechet"""
```

Comment:
Invalid docstring format. Use `coala --ci` and format the code using [black](https://github.com/psf/black)